### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Isles22**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+        arr.append(Person(line["name"], line["age"]))
+
+    for obj in people:
+        for human in arr:
+            if obj["name"] == human.name:
+                for human_family in arr:
+                    if obj.get("wife") == human_family.name:
+                        human.wife = human_family
+                    if obj.get("husband") == human_family.name:
+                        human.husband = human_family
```
Comment (DmytroSvirsa) [inline app/main.py:23]: that`s will work but slowly, let`s make it less nested, I mean not nested at all

File: app/main.py [hunk 2]
```diff
@@ -1,8 +1,24 @@
...
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    arr = []
```
Comment (DmytroSvirsa) [inline app/main.py:12]: not self-explaining variable name

**Review by DmytroSvirsa — CHANGES_REQUESTED** [review]

**Comment by Isles22** [comment]
Solution fixed

чт, 10 нояб. 2022 г. в 10:31, Dmytro Svirsa ***@***.***>:

**Review by OleksandrPozden — APPROVED** [review]
well done!

**Pull request closed without merging**